stage1/init/init.go

```diff
@@ -295,6 +316,167 @@ func forwardedPorts(pod *Pod) ([]networking.ForwardedPort, error) {
 	return fps, nil
 }
 
+func parseCgroups() (map[int][]string, error) {
+	f, err := os.Open("/proc/cgroups")
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	sc := bufio.NewScanner(f)
+
+	sc.Scan()
```

Comment:
Add a comment to say that `sc.Scan()` is to skip the first line because it is a comment?
```
#subsys_name	hierarchy	num_cgroups	enabled
```